src/components/admin/providers/provider-form.tsx

```
@@ -0,0 +1,604 @@
+    const [documents, setDocuments] = useState<Record<string, File | File[] | null>>({
+        workingContract: null,
+        kyc: null,
+        cv: [],
+        declarationForm: null,
+        professionalDocs: [],
+    });
+    const [interventionsOffered, setInterventionsOffered] = useState<string[]>([]);
+    // Group interventions by service
+    const interventionsByService = useMemo(() => {
+        const groups: Record<string, Intervention[]> = {};
+        if (services && services.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)